Packs/LookoutCASB/README.md

```diff
@@ -0,0 +1,14 @@
+### Lookout CASB Integration
```

Comment:
This content is a copy of the integration readme. Please delete this content and add a PACK readme (required) that talks about the value of the pack, what it does, what Lookout does, links to your site, etc. Let us know if you need help or example.

Reply:
Sure will change it.

Reply:
please update this file